.codecov.yml

```diff
@@ -6,4 +6,4 @@ coverage:
 
     patch:
       default:
-        target: 10%
+        target: 0%
```

Comment:
Is this change intentional?

Reply:
Yes, since we haven't covered these util classes yet, I discussed with @qilinkang and temporarily set it to zero to allow the checks to pass.